plugins/org.yakindu.sct.model.stext/src/org/yakindu/sct/model/stext/validation/STextValidationMessages.java

```diff
@@ -61,4 +61,6 @@
 	public static final String CONTRADICTORY_ANNOTATIONS = "Some annotations (%s) have contradictory effects.";
 	public static final String BAD_EVENT_NAMES = "'%s' is already used as name of an internal event.";
 	public static final String SYNC_OUTGOING_TRIGGER = "Triggers and guards on a synchronization's outgoing transitions will be ignored.";
+	public static final String ALWAYS_TRUE_TRANSITION_USED = "Transition with '%s' should have the lowest priority, otherwise transitions with lower priority will never be taken.";
```

Comment:
This message might be confusing. The highest priority has the lowest number... Additionally state how to fix the problem.

So we should use a alternative message like: "Transition with '%s' should be evaluate as the last transition of the state as following transitions won't be evaluated. Change the transition order (in the Properties view)."